packages/DropdownMenu/stories/examples/DropdownMenuExample.js

```diff
@@ -34,7 +37,8 @@ const DropdownMenuExample = () => {
               body="Lorem ipsum dolor amet vexillologist tacos selvage narwhal butcher twee ethical hot chicken."
               confirmLabel="Delete filter"
               heading="Delete filter?"
-              onConfirm={handleConfirm(onCloseMenu)}
+              onConfirm={handleConfirm}
+              onClose={handleCloseConfirm(onCloseMenu)}
```

Comment:
I think adding an `isPending` example to the DropdownMenu would be helpful

Reply:
I'll add to the showcase